kibana/templates/configmap-dashboardimport.yaml

```diff
@@ -0,0 +1,61 @@
+{{- if .Values.dashboardImport.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kibana.fullname" . }}-importscript
+  labels:
+{{ include "kibana.labels" . | indent 4 }}
+data:
+  dashboardImport.sh: |
+    #!/usr/bin/env bash
+    #
+    # kibana dashboard import script
+    #
```

Comment:
a `set -euo pipefail` up here will make sure that the script fails properly if any of the commands fail.

http://redsymbol.net/articles/unofficial-bash-strict-mode/